.github/workflows/pull_request.yml

```diff
@@ -46,9 +52,52 @@ jobs:
       - uses: actions/cache@v1
         with:
           path: /tmp/release-snacks-case-studies/
-          key: release-snacks-case-studies
+          key: release-snacks-case-studies-${{ runner.os }}-${{ hashFiles('**/flyte_tests_manifest.json') }}-${{ github.sha }}
       - name: Generate Serialize proto
         env:
           REGISTRY: "ghcr.io/flyteorg"
         run: |
-          make -C cookbook/case_studies serialize
\ No newline at end of file
+          make -C cookbook/case_studies serialize
+          mkdir -p cookbook/release-snacks /tmp/release-snacks-case-studies/
+          make -C cookbook archive
+          cp -r cookbook/release-snacks/* /tmp/release-snacks-case-studies/
+  sandbox:
+    name: Register & Serialize example
+    runs-on: ubuntu-latest
+    needs: [core-build, case_studies-build, integrations-build]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
+      - uses: actions/cache@v1
+        with:
+          path: /tmp/release-snacks-core/
+          key: release-snacks-core-${{ runner.os }}-${{ hashFiles('**/flyte_tests_manifest.json') }}-${{ github.sha }}
+      - uses: actions/cache@v1
+        with:
+          path: /tmp/release-snacks-case-studies/
+          key: release-snacks-case-studies-${{ runner.os }}-${{ hashFiles('**/flyte_tests_manifest.json') }}-${{ github.sha }}
+      - uses: actions/cache@v1
+        with:
+          path: /tmp/release-snacks-integrations/
+          key: release-snacks-plugins-${{ runner.os }}-${{ hashFiles('**/flyte_tests_manifest.json') }}-${{ github.sha }}
+      - uses: unionai/flytectl-setup-action@v0.0.1
+        with:
+          version: '0.1.18'
+      - name: "Setup env"
+        run: mkdir ~/.flyte
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
+          ref: ${{ steps.bump-version.outputs.tag }}
+      - name: Copy artifacts
+        run: |
+          mkdir -p cookbook/release-snacks
+          cp -r /tmp/release-snacks-core/* cookbook/release-snacks/
+          cp -r /tmp/release-snacks-case-studies/* cookbook/release-snacks/
+          cp -r /tmp/release-snacks-integrations/* cookbook/release-snacks/
+      - name: setup sandbox
+        run: make setup
```

Comment:
I want to start thinning the Makefile and simplifying it... it was just a stop-gap until we have something more elegant. And, thanks to you, we now do!

```suggestion
        run: flytectl sandbox start
```